src/manage/project-advanced.md

```diff
@@ -1,64 +1,97 @@
-# How to Share, Transfer or Delete Project
+# How to Share, Transfer or Delete Projects
 [[toc]]
 
-## Share a project
+## Share projects and manage user access
+You can share your <MainPlatformName /> project with others by inviting them to be [a member or a guest](./permissions/#workspace-members-and-guests) in your workspace. You can also make your <MainPlatformName /> project accessible to everyone by making it [public](#make-your-project-public-private).
```

Comment:
I think it might be a bit abstract to mention a "Share button" and "share dialogue" as the intention of this how to page is to basically guide them to the share button (even if they don't know the difference between a member and a guest yet)